server/src/paypal-client.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+/**
+ *
+ * PayPal Node JS SDK dependency
+ */
+const checkoutNodeJssdk = require('@paypal/checkout-server-sdk');
+
+/**
+ *
+ * Returns PayPal HTTP client instance with environment that has access
+ * credentials context. Use this instance to invoke PayPal APIs, provided the
+ * credentials have access.
+ */
+function client() {
+  return new checkoutNodeJssdk.core.PayPalHttpClient(environment());
+}
+
+/**
+ *
+ * Set up and return PayPal JavaScript SDK environment with PayPal access credentials.
+ * This sample uses SandboxEnvironment. In production, use LiveEnvironment.
+ *
+ */
+function environment() {
+  let clientId = process.env.PAYPAL_CLIENT_ID;
+  let clientSecret = process.env.PAYPAL_CLIENT_SECRET;
+
+  return new checkoutNodeJssdk.core.LiveEnvironment(
+    clientId, clientSecret
+  );
+}
+
+async function prettyPrint(jsonData, pre=""){
+  let pretty = "";
```

Comment:
What is this variable? Its hard to understand what  this variable does just by the name 'pretty'

Reply:
I didn't make this code since this code was made by PayPal so, I'm also not sure what it is but, It seems like making json data to "pretty" looking string.

Reply:
> What is this variable? Its hard to understand what this variable does just by the name 'pretty'

I didn't make this code since this code was made by PayPal so, I'm also not sure what it is but, It seems like making json data to "pretty" looking string.